test/runner.js

```diff
@@ -36,21 +35,9 @@ if ( program.grep ) {
 	mocha.grep( new RegExp( program.grep ) );
 }
 
-if ( process.env.CIRCLECI ) {
```

Comment:
It would be nice to get rid of it, but it looks like some tests are still bombing without it.